apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -225,6 +227,8 @@ type K8ssandraClusterList struct {
 type CassandraClusterTemplate struct {
 	DatacenterOptions `json:",inline"`
 
+	Meta EmbeddedObjectMeta `json:"metadata,omitempty"`
```

Comment:
This is exposing `name ` and `namespace` as well at cluster-level, which imho doesn't make sense.